pkg/shell/state.tsx

```diff
@@ -55,6 +55,14 @@ export interface ShellStateEvents {
     connect: () => void;
 }
 
+// NOTE - this is defined in pkg/lib/notifications.js and should be
+// imported from there once that file has been typed.
+//
+export interface PageStatus {
```

Comment:
OOI -- could we just create a pkg/lib/notifications.d.ts and put that there, as an intermediate step? Or would that require moving more chairs around the deck?

(Not blocking or a veto, just learning)

Reply:
I think we could!

Reply:
Hmm, let's port it to TS right away, separately.